.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2
+jobs:
+  build:
+    parallelism: 4
```

Comment:
To clarify, I worry about it trying to run the tests and `make bootstrap` at the same time. Have you run into those issues?